README.md

```diff
@@ -335,6 +335,34 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+## Test helpers
```

Comment:
@maurogeorge could you move this to an article on our [new docs](https://github.com/rails-api/active_model_serializers/tree/master/docs) and maybe but a reference here in the README?
Maybe into the "How to" section explaining hot to assert serializer.